console/src/routes/advanced/+page.svelte

```diff
@@ -0,0 +1,189 @@
+<script lang="ts">
+	import TableHeadItem from '$lib/tables/TableHeadItem.svelte';
+	import TableDataCell from '$lib/tables/TableDataCell.svelte';
+	import HopsNav from '$lib/nav/HopsNav.svelte';
+	import type { PageData } from './$types';
+	import Loading from '$lib/components/Loading.svelte';
+	import type { EventTable } from '$lib/events/api';
+
+	//Set default tab
+	let tab = 'Events';
+
+	export let data: PageData;
+
+	// Event logs table data
+	$: tableData = data.tableData;
+	$: ago = data.ago;
+	$: loading = data.tableData === undefined;
+
+	// Dummy Pipeline data
+	let pipelineData = [
+		{
+			name: 'Pipeline Name #1',
+			metadata: 'Metadata',
+			pastStatus: ['success', 'failure', 'success', 'success', 'failure']
+		},
+		{
+			name: 'Pipeline Name #2',
+			metadata: 'Metadata',
+			pastStatus: ['failure', 'failure', 'failure', 'success', 'failure']
+		},
+		{
+			name: 'Pipeline Name #3',
+			metadata: 'Metadata',
+			pastStatus: ['success', 'failure', 'failure', 'success', 'failure']
+		}
+	];
+
+	$: activeRow = tableData?.[0];
+
+	function setActiveRow(row: EventTable) {
+		activeRow = row;
+	}
+</script>
+
+<svelte:head>
+	<title>Advanced</title>
+	<meta name="description" content="Trigger task flows" />
+</svelte:head>
+
+{#if loading}
+	<Loading />
+{:else}
+	<div class="dark:bg-grain bg-cover bg-norepeat bg-fixed min-h-screen">
+		<HopsNav />
+
+		<!--Page title-->
+		<div class="md:pl-20 md:pr-20 mx-8 md:mx-auto mt-12 pb-4">
+			<h1 class="text-left text-5xl font-normal mb-2 dark:text-white">Advanced</h1>
+			<h2 class="text-2xl mb-4 text-grey dark:text-white dark:text-opacity-50">
+				Showing all events since {ago}
+			</h2>
+
+			<!--Tabs-->
```

Comment:
Reason this is commented out is that the next story is to add this functionality in. Makes it (arguably) easier than stripping it out and then finding it again.